<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionHistoryWidget</class>
 <widget class="QWidget" name="ConnectionHistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection history</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="EnhancedTableView" name="tableView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Delete log records</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonDeleteOlder">
              <property name="text">
               <string>Delete older records</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteAll">
              <property name="text">
               <string>Delete all records</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Automatic log record deletion configuration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkBoxAutomaticDeletion">
              <property name="text">
               <string>Automatic deletion after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxDays">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>days</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteSetting">
              <property name="text">
               <string>Update setting</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedTableView</class>
   <extends>QTableView</extends>
   <header>App.Widget.View</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAutomaticDeletion</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBoxDays</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
